service/src/main/java/uk/gov/hmcts/reform/fpl/service/robotics/RoboticsDataService.java

```diff
@@ -64,9 +64,8 @@ public RoboticsData prepareRoboticsData(final CaseData caseData, final Long case
             .internationalElement(hasInternationalElement(caseData.getInternationalElement()))
             .allocation(isNotEmpty(caseData.getAllocationProposal())
                 && isNotBlank(caseData.getAllocationProposal().getProposal())
-                ? caseData.getAllocationProposal().getProposal() :  null)
-            .issueDate(isNotEmpty(caseData.getDateSubmitted())
-                ? formatLocalDateToString(caseData.getDateSubmitted(), "dd-MM-yyyy") : "")
+                ? caseData.getAllocationProposal().getProposal() : null)
+            .issueDate(formatDate(caseData.getDateSubmitted(), "dd-MM-yyyy"))
```

Comment:
Do we already have a constant for that format in the DateFormatterHelper? If not do you think it would be sensible to add one?

Reply:
It is an option, but this is format required by external service and used only by it. If it is common format then can be used in other places and changed when needed, which o course cause test to fail which is good. Problematic is also naming of such constant as we have already cople of them, how to name them DATE_SHORT_MONTH, it is already there for different format, ROBOTICS_DATE does not really fit into common class?

Reply:
Fair enough, it definitely does appear to be a different format to what we would normally use so I am happy for it to not be a constant in that class, will sonarqube complain about it not being a constant in this class though?

Reply:
sonar complains when there are 3 repetition of string i guess